include/dynamatic/Analysis/ControlDependenceAnalysis.h

```diff
@@ -0,0 +1,82 @@
+//===- ControlDependenceAnalysis.h - Control dependence analyis *--- C++-*-===//
+//
+// Dynamatic is under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Declares some utility functions useful to analyzing the control dependencies
+// between basic blocks of the CFG.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef DYNAMATIC_ANALYSIS_CONTROLDEPENDENCEANALYSIS_H
+#define DYNAMATIC_ANALYSIS_CONTROLDEPENDENCEANALYSIS_H
+
+#include "dynamatic/Support/LLVM.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Pass/AnalysisManager.h"
+#include <optional>
+
+#define DEBUG_TYPE "CONTROL_DEPENDENCE_ANALYSIS"
+
+using namespace mlir;
```

Comment:
Missed this during the first review, but never include namespaces in header files, this can create hard to resolve name conflicts and pollutes the namespace in source files with no way of opting out.